modules/apps/headless/headless-discovery/headless-discovery-web/webpack.config.js

```diff
@@ -40,7 +40,7 @@ const config = {
 	},
 	output: {
 		filename: buildName,
-		path: path.resolve('src/main/resources/META-INF/resources/dist'),
+		path: path.resolve('./build/node/packageRunBuild/resources/'),
```

Comment:
yes, it would, we just wanted to make it like [the one we have in `frontend-js-web`](https://github.com/liferay/liferay-portal/blob/8eb0ae798c7f743c16b365949c83563afe5b6651/modules/apps/frontend-js/frontend-js-web/webpack.config.js#L38)

Reply:
Sure. Just following the same pattern I've found in other webpack config files.  I could change it in a moment if you think it is better that way. Thank you very much for all your help!

Reply:
Nah, I don't think it matters. I just brought it up because it stood out a bit to me that you'd gone from `src/` to `./build/` instead of `src/` to `build/`.